scene/gui/color_picker.cpp

```diff
@@ -399,7 +399,7 @@ void ColorPicker::_screen_input(const InputEvent &ev)
 		Image img =r->get_screen_capture();
 		if (!img.empty())
 			last_capture=img;
-			r->queue_screen_capture();
```

Comment:
Ah sorry I should have commented a few lines down, I didn't remove that line I just changed its indentation to show that it's not included in the if block. What I don't know is if it's meant to or if the extra indentation is a mistake.